src/components/FeatureInfoPopup.js

```diff
@@ -167,7 +167,13 @@ class FeatureInfoPopup extends Component {
                           >
                             {propertyName.name}
                           </label>
-                          <span>{feature.properties[propertyName.key] + ''}</span>
+                          {
+                            propertyName.isUrl===true
+                            ? <a href={feature.properties[propertyName.key]}>
```

Comment:
I think it's more convenient for users to add target="_blank" into the a tag

Reply:
Good idea! :)